Cargo.toml

```diff
@@ -5,13 +5,17 @@ authors = ["Michael Kirk <michael.code@endoftheworl.de>"]
 edition = "2021"
 
 [dependencies]
-approx = "0.4.0"
-geo = "*"
-include_dir = { version = "0.6", features = ["search"] }
+approx = "0.5.0"
+geo = "0.19.0"
+include_dir = { version = "0.7.2", features = ["glob"] }
 log = "0.4.14"
-serde =  { version = "*", features = ["derive"] }
-serde-xml-rs = "*"
+serde =  { version = "1.0.105", features = ["derive"] }
+serde-xml-rs = "0.5.0"
 wkt = { version = "0.10.0", features = ["geo-types", "serde"] }
 
 [dev-dependencies]
-pretty_env_logger = "*"
+pretty_env_logger = "0.4.0"
+
+[patch.crates-io]
+geo-types = { git = 'https://github.com/georust/geo', branch = 'mkirk/update-approx' }
```

Comment:
BTW - this patch section is the circular dependency dance that we can avoid if we merge the two repos.